openmdao/drivers/tests/test_analysis_errors.py

```diff
@@ -26,7 +26,7 @@
     'PSQP',         # fails nominal with: (4) Maximum constraint value is less than or equal to tolerance
 }
 
-@use_tempdirs
+# @use_tempdirs
```

Comment:
was commenting this out meant to be temporary?

Reply:
Fixed, thanks